Competition2017/src/competition/subsystems/fuel_launcher/commands/MaintainShooterPowerCommand.java

```diff
@@ -0,0 +1,25 @@
+package competition.subsystems.fuel_launcher.commands;
+
+import competition.subsystems.shooter.SideShooterSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class MaintainShooterPowerCommand extends BaseCommand {
+
+    final SideShooterSubsystem sideShooter;
+
+    public MaintainShooterPowerCommand(SideShooterSubsystem sideShooter) {
+        this.sideShooter = sideShooter;
+        this.requires(this.sideShooter);
+    }
+
+    @Override
+    public void initialize() {
+    
+    }
+    
+    @Override
+    public void execute() {
+        sideShooter.updateTelemetry();
```

Comment:
Given how the CANTalons work (no new updates mean just keep doing what you're doing), a no-op is actually a "keep doing what you are doing."  

(As a total strawman, if you saw "DoNothingToShooterCommand" on the SmartDashboard, but saw the shooter spinning rapidly under PID control, you would be rightly confused).

@WasabiFan That said, I'm definitely open to better names for anything - what do you think makes more sense here?